src/components/Recent.tsx

```diff
@@ -113,12 +111,24 @@ export function RecentCard(props: RecentCardProps): React.ReactElement {
                 <Link
                   to={{
                     pathname: resource,
-                    search:
-                      onFilter !== false
-                        ? `filter=${JSON.stringify(
```

Comment:
@TahaKhanAbdalli  - I would prefer it if the following filters were passed through in the props, please.  Then the logic is stored where the item is declared, rather than being `inferred` from the resource type.